.github/workflows/build.yml

```diff
@@ -13,14 +13,17 @@ jobs:
       fail-fast: true
       matrix:
         os:
-          - macos-12
-          - macos-11
+          - macos-latest
         build_type:
           - Debug
           - Release
         qt:
           - version: "5.15.2"
             requested: "5.15"
+            architecture: "x86_64"
+          - version: "6.4.*"
+            requested: "6.4"
+            architecture: "arm64"
```

Comment:
now that you did this, I'm actually thinking I should actually have done this to GammaRay too, but actually it would be nicer if it was "x86_64;arm64" as that would build both, maybe you should try this here too.

Reply:
For Qt 6, good idea.

Reply:
`x86_64;arm64` didn't work, I've worked around it by making adding another Qt 6 entry, one for each architecture.
```
          - version: "5.15.2"
            requested: "5.15"
            architectures: "x86_64"
          - version: "6.4.*"
            requested: "6.4"
            architectures: "x86_64"
          - version: "6.4.*"
            requested: "6.4"
            architectures: "arm64"
```

Reply:
Maybe you forgot the quotes? because `;` char can finish the cmd line

Reply:
Commit 8d935b3d018daec4de858fa8add32c65bf97a8ba looked like this:
```
            architectures: "x86_64;arm64"
```
```
          -DCMAKE_OSX_ARCHITECTURES=${{ matrix.qt.architectures }}
```

Reply:
would this work? -DCMAKE_OSX_ARCHITECTURES="${{ matrix.qt.architectures }}"

Reply:
That worked! Thanks